controllers/storage/minio.go

```diff
@@ -127,8 +128,25 @@ func (m *MinIOReconciler) Reconcile() (*lcm.CRStatus, error) {
 	return minioUnknownStatus(), nil
 }
 
-func createDefaultBucket() error {
-	panic("implement me")
+func (m *MinIOReconciler) minioInit() error {
+	accessKey, secretKey, err := m.getCredsFromSecret()
+	if err != nil {
+		return err
+	}
+	endpoint := m.getServiceName() + "." + m.HarborCluster.Namespace
+
+	m.MinioClient, err = GetMinioClient(endpoint, string(accessKey), string(secretKey), DefaultRegion, false)
+	if err != nil {
+		return err
+	}
+
+	exists, err := m.MinioClient.IsBucketExists(DefaultBucket)
+	if err != nil || exists {
```

Comment:
If it has already been there, why not pick up it again? What's the meaning of returning an error here?

Reply:
the funcation of `minioInit` just needs to create a default bucket for harbor to use. if there has already been created, we can just skip it, in which case the `error` is always equal to nil.

Reply:
ok